packages/amplify-migration-tests/src/__tests__/migration_tests/lambda-layer-migration/layer-migration.test.ts

```diff
@@ -28,14 +28,14 @@ describe('test lambda layer migration flow introduced in v5.0.0', () => {
   let projRoot: string;
   let versionToMigrateFrom: string;
 
-  beforeAll(async () => {
+  beforeEach(async () => {
```

Comment:
this change, and similar changes to the files in src/__tests__/ are done to ensure that "createNewProjectDir(..)" is the first thing that happens in a test.

There were only a handful of tests that needed this, specifically - migration tests.